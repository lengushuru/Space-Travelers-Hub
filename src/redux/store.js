import { configureStore } from '@reduxjs/toolkit';
import mission001Reducer from './missions/missionsSlice';

const store = configureStore({
  reducer: {
    mission001: mission001Reducer,
  },
});

export default store;
