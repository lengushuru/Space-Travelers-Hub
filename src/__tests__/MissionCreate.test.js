import React from 'react';
import { cleanup, render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../redux/store';
import MissionCreate from '../redux/missions/MissionCreate';

afterEach(cleanup);

describe('missions tests', () => {
  it('MissionCreate to match Snapshot', () => {
    const missionCreate = render(<Provider store={store}><MissionCreate /></Provider>);
    expect(missionCreate).toMatchSnapshot();
  });
});