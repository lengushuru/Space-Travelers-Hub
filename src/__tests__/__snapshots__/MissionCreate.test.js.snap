// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`missions tests MissionCreate to match Snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container"
      >
        <div
          class="mission-container title_hieght"
        >
          <h2
            class="mission_name title_hieght"
          >
            Missions
          </h2>
          <h2
            class="mission_description title_hieght"
          >
            description
          </h2>
          <h2
            class="mission_name title_hieght"
          >
            status
          </h2>
          <p
            class="mission_name title_hieght"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container"
    >
      <div
        class="mission-container title_hieght"
      >
        <h2
          class="mission_name title_hieght"
        >
          Missions
        </h2>
        <h2
          class="mission_description title_hieght"
        >
          description
        </h2>
        <h2
          class="mission_name title_hieght"
        >
          status
        </h2>
        <p
          class="mission_name title_hieght"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
